tfjs-backend-webgpu/src/shader_preprocessor.ts

```diff
@@ -115,6 +116,10 @@ export function makeShader(
     uniformDeclaration += 'int size; ';
   }
 
+  if (program.reshapeDispatch) {
```

Comment:
You can always declare `dispatchSize` and remove the checking `if (program.reshapeDispatch)`.